tests/aws/services/lambda_/functions/lambda_echo_json_body.py

```diff
@@ -18,8 +18,18 @@ def handler(event, context):
 
 import json
 
-
 def handler(event, context):
-    # Just print the event that was passed to the Lambda
-    print(json.dumps(event))
-    return json.loads(event["body"])
+    print(f"Received event: {json.dumps(event)}")
+
+    response_body = {
+        "message": "Hello from Lambda!",
+        "input": event
+    }
+    return {
+        "isBase64Encoded": False,
+        "statusCode": 200,
+        "headers": {
+            "Content-Type": "application/json"
+        },
+        "body": json.dumps(response_body)
+    }
```

Comment:
Is there a reason you are passing in these extra fields? As mentioned above, this lambda is used a lot throughout the code and I'd be hesitant to change it's functionality much (if at all 🫠)